lms/djangoapps/discussion/django_comment_client/base/tests.py

```diff
@@ -1726,35 +1726,6 @@ def setUpTestData(cls):
         cls.student.roles.add(Role.objects.get(name="Student", course_id=cls.course.id))
         CourseAccessRoleFactory(course_id=cls.course.id, user=cls.student, role='Wizard')
 
-    @patch('eventtracking.tracker.emit')
-    @patch('openedx.core.djangoapps.django_comment_common.comment_client.utils.requests.request', autospec=True)
```

Comment:
@feanil as we are mocking the request but ignoring in the arguments. When we create our own request in test mocked one is called and its always successful in python2 but when we come to python3 it fails as no return value is set and default is none. I tried to remove it and check if the test passes or not but it fails which means test is broken in py2. In the same file for some other broken tests with same error you suggested and we removed a couple of such tests from another class. So i removed this too.

Reply:
This is really good context, please put it in the commit message so that if people are looking at the history they can easily understand _why_ things happened.

Reply:
done.